public/app/core/utils/explore.test.ts

```diff
@@ -187,15 +187,15 @@ describe('hasNonEmptyQuery', () => {
 });
 
 describe('getTimeRange', () => {
-  describe('should flip from and to when from is after to', () => {
+  describe('should not flip from and to when from is after to', () => {
     const rawRange = {
       from: 'now',
       to: 'now-6h',
     };
 
     const range = getTimeRange('utc', rawRange, 0);
 
-    expect(range.from.isBefore(range.to)).toBe(true);
+    expect(range.from.isBefore(range.from)).toBe(false);
```

Comment:
```suggestion
    expect(range.from.isBefore(range.to)).toBe(false);
```

Don't think we meant to change this right?

Reply:
yes 🤦‍♀️